### DISCUSSION TITLE
Question to all Contributors.

### DISCUSSION THREAD

**Discussion by Drapersniper** [Cog-Creators/Red-DiscordBot#4171] [General]
If your name is in this list https://github.com/Cog-Creators/Red-DiscordBot/graphs/contributors

Could you please reply to this issue with your discord name so that we can look into getting you the contributor role in the discord server?

**Comment by douglascdev** [comment]
Douglas#6849

**Comment by PythonTryHard** [comment]
Hitori Tori#5630 (if we're still open)
